schemas/layer-2.cue

```diff
@@ -1,84 +1,131 @@
     "shared-threats"?: [...#Mapping] @go(SharedThreats)
+    // zero or more mapping references to capabilities defined in this or other Layer2 frameworks or standards
     "shared-capabilities"?: [...#Mapping] @go(SharedCapabilities)
 }
 
-// Resuable types //
-
 #Metadata: {
+    // unique identifier for the Layer2 collection
     id: string
+    // name for the Layer2 collection
     title: string
+    // description of the Layer2 collection
     description: string
     version?: string
+    // timestamp of the last modification to the Layer2 collection
     "last-modified"?: string @go(LastModified)
+    // Applicability categories are facets that apply to the controls in this Layer2 collection. These facet values can be used to filter which controls from Layer2 are used in Layer 3 or Layer 4.
     "applicability-categories"?: [...#Category] @go(ApplicabilityCategories)
+    // List of applicable references to Layer 1 guidance, technical capabilities and threats that inform the Layer2 collection
     "mapping-references"?: [...#MappingReference] @go(MappingReferences)
 }
 
+// Category defines a facet that applies to the AssessmentRequirements in Controls. Categories typically define "levels" of compliance, for example the low, moderate and high baselines within the NIST 800-53 controls.
 #Category: {
+    // unique identifier for the category
     id: string
+    // name of the category
     title: string
+    // description of the category
     description: string
 }
 
+// ControlFamily is a collection of security controls that are grouped together based on a common theme or purpose. Control families are used to organize and categorize security controls within a Layer2 framework or standard.
 #ControlFamily: {
+    // name of the control family
     title: string
+    // description of the control family
     description: string
+    // the Controls that are part of this ControlFamily
     controls: [...#Control]
 }
 
+// Controls are the specific guardrails that organizations put in place to protect their information systems. They are typically informed by the best practices and industry standards which are produced in Layer 1. Controls are typically developed by an organization for its own purposes, or for general use by industry groups, government agencies, or international standards bodies.
 #Control: {
+    // unique identifier for the control
     id: string
+    // name of the control
     title: string
+    // the intended outcome of applying the control
     objective: string
+    // the assessment requirements that are used to determine if the control is met
     "assessment-requirements": [...#AssessmentRequirement] @go(AssessmentRequirements)
-
-    "guideline-mappings"?: [...#Mapping] @go(GuidelineMappings)
+    // references to layer 1 guidance or standards that inform the control
+    "guidance-mappings"?: [...#Mapping] @go(GuidanceMappings)
```

Comment:
Note the rename here and the fact that baseline will need an update either way, see https://github.com/ossf/security-baseline/blob/main/baseline/OSPS-LE.yaml#L22